draft-irtf-cfrg-vdaf.md

```diff
@@ -4574,7 +4574,13 @@ exposing" a measurement by including it too many times in a single batch or
 across multiple batches. It is RECOMMENDED that the nonce generated by the
 Client be used by the Aggregators for replay protection.
 
-## Requirements for the Aggregation Parameters
+## Requirements for the Public Share
+
+The Aggregators MUST verify that have both received the same public share from
+the Client. It is sufficient, for example, to exchange a hash of the pubic
+share over a secure channel.
```

Comment:
Do we satisfy this requirement in DAP?

Reply:
Yes, since public share is bound to HPKE decryption.

Reply:
Maybe I'm missing something: in DAP , the client separately HPKE encrypts input shares to either aggregator's respective key. A malicious could therefore use distinct public shares and construct distinct AADs, no?

Reply:
The risk that the Poplar paper talks about when the aggregators don't have a consistent view of the public parameters is that the malicious client could control the output of the DPF at two inputs, instead of one, like we want. This happens because choosing different group element correction words for one level, and giving each aggregator a different one, gives the client an extra degree of freedom it shouldn't have.

Since we're considering a malicious client when we consider attempts to break extractability in this manner, we don't need to protect this report's privacy, and thus the only goal that matters is robustness. For robustness, we get to assume that both aggregators are honest, and thus we can assume that the leader will distribute the public share to the helper correctly, without considering the HPKE AAD.

Including the public share in the HPKE AAD is probably more relevant for protecting against privacy attacks, as we wouldn't want to let a rogue leader pass an honest client's input share with a tampered public share.